storage/innobase/lock/lock0lock.cc

```diff
@@ -7304,6 +7306,26 @@ and less modified rows. Bit 0 is used to prefer orig_trx in case of a tie.
         }
         snprintf(buf, sizeof buf, rollback_msg, victim_pos);
         print(buf);
+
+        if (srv_print_all_deadlocks)
+        {
+          rewind(lock_latest_err_file);
```

Comment:
a `rewind` + `fseek` + `ftell` seems to be messing around a little too much. What's the minimal required?